apps/www/components/Article/metadata.js

```diff
@@ -72,6 +72,26 @@ const getJSONLDs = (meta) => {
   }
 }
 
+function getCitationMetaData(meta) {
+  const authors = meta.contributors
+    .filter((contributor) => contributor?.kind?.includes('Text'))
+    .map((contributor) => {
+      const parts = contributor.name.split(' ')
+      // Convert the name into the following format `Surname, Given Name(s)`
+      return `${parts[parts.length - 1]}, ${parts
+        .slice(0, parts.length - 1)
+        .join(' ')}`
+    })
+    .join('; ')
+
+  return {
+    citation_title: meta.title,
+    citation_journal_title: PUBLIC_BASE_URL,
```

Comment:
What's the thinking behind putting in the url here instead of `Republik Magazin` or something like this?